Sprint-1/exercises/initials.js

```diff
@@ -4,3 +4,6 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string.
 // This should produce the string "CKJ", but you must not write the characters C, K, or J in the code of your solution.
+
+let initials = firstName[0] + middleName[0] + lastName[0];
```

Comment:
Great showcasing of String manipulation, well done! 🎉

As you dive deeper into String methods, you will see that there are pretty much unlimited ways you can approach to solving a problem. You can get really creative and even produce your own unique style of writing code. That is the beauty of programming, it is very creative as well as very analytical.